Configuration/Settings.yaml

```diff
@@ -2,3 +2,5 @@ Wegmeister:
   LanguageRedirect:
     # Add mappings for language codes if you use some different codes than the default ones.
     languageCodeOverrides: {}
+    # configure the name of your frontend language cookie name
+    feLanguageCookieName: _fe_language
```

Comment:
Use `null` as default here, because one has to implement the cookie handling for this to work. This will prevent the backend from trying to read cookies that are missing.
If you want, you can add some more information on this in the readme, but I'm also fine with the current state.